### PR TITLE
Add informational advice on MCE substitution

### PR DESCRIPTION
**Pull request by cabo**
(how to substitute multi-character escapes in order to obtain an
I-Regexp).

### CODE REVIEW

File: draft-bormann-jsonpath-iregexp.md [hunk 1]
```diff
@@ -315,6 +315,62 @@ escapes, all regular expressions validate against the ABNF in {{iregexp-abnf}}.
...
+
+* Replace any multi-character escape outside character classes as
+  in {{tbl-mce}}.
+* For any multi-character escape that show a charClassEsc (and not a
+  charClassExpr) in the second column of {{tbl-mce}}, replace them
+  inside the charClassExpr of the regexp as per {{tbl-mce}}.
+* For multi-character escape that do show a charClassExpr in the second column
+  of {{tbl-mce}} ("CCE2"), replace them inside charClassExpr of the
+  regexp ("CCE1") as follows:
+  * Examine for both charChlassExpr whether it is negative (has a "^"
```
Comment (timbray) [inline draft-bormann-jsonpath-iregexp.md:347]: typo

File: draft-bormann-jsonpath-iregexp.md [hunk 2]
```diff
@@ -315,6 +315,62 @@ escapes, all regular expressions validate against the ABNF in {{iregexp-abnf}}.
...
+advice for obtaining an I-Regexp.
+
+These rules do not address `\w` or `\S` in negative charClassExpr,
+i.e., in charClassExpr that include the optional "^" at the start,
+which however can be substituted by employing character class
+subtraction.
+
+* Replace any multi-character escape outside character classes as
+  in {{tbl-mce}}.
+* For any multi-character escape that show a charClassEsc (and not a
```
Comment (timbray) [inline draft-bormann-jsonpath-iregexp.md:341]: that contains a

File: draft-bormann-jsonpath-iregexp.md [hunk 3]
```diff
@@ -315,6 +315,62 @@ escapes, all regular expressions validate against the ABNF in {{iregexp-abnf}}.
...
+    charClassExpr, see above), more complex transformation may be required.
+    <!-- which sometimes requires character class subtraction,
+     (e.g., `[^-\w_]` means `[\p{P}\p{Z}\p{C}-[-_]]`, which requires
+    taking apart \p{P} and within that \p{Pc} and \p{Pd}; but,
+    really, [^-A-Za-z0-9_] may be what was actually meant).  -->
+
+| MCE  | I-Regexp equivalent | charClassEsc |
+| `\d` | \p{Nd}              | t            |
+| `\s` | [ \t\n\r]           | f            |
+| `\w` | [^\p{P}\p{Z}\p{C}]  | f            |
```
Comment (timbray) [inline draft-bormann-jsonpath-iregexp.md:363]: So unreliable, I hate to encourage it

**Comment by cabo** [comment]
Given #8, I think we'll keep this as useful information, but don't include it in the specification.

**Pull request closed without merging**